actions/post_attachment.py

```diff
@@ -0,0 +1,63 @@
+import json
+
+from six.moves.urllib.parse import urlencode
+import requests
+
+from st2common.runners.base_action import Action
+
+__all__ = [
+    'PostMessageAction'
+]
+
+
+class PostMessageAction(Action):
+    def run(self, attachment, username=None, icon_emoji=None, icon_url=None, channel=None,
+            disable_formatting=False, webhook_url=None):
+        config = self.config.get('post_message_action', {})
+        username = username if username else config['username']
+        icon_emoji = icon_emoji if icon_emoji else config.get('icon_emoji', None)
+        icon_url = icon_url if icon_url else config.get('icon_url', None)
```

Comment:
In README and in config.schema then I don't think icon_url is a config option - so either we should add it to those files, or don't try and take from config.

Reply:
@amanda11
Hmm interesting your correct with the ``icon_url``, this was copied from the ``post_message`` action so I updated the PR to remove that line from both the new ``post_attachment`` and existing ``post_message`` actions.

As as far as a new section it would be an identical config with identical settings so I think it is better to utilize the same config as to not repeat it.  I did however add a note in the read me to call this out to help mitigate confusion.  Let me know your thoughts on the updates.